app.js

```diff
@@ -16,6 +18,8 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+app.use('/api', router);
+
 app.get('*', (request, response) => {
   const welcome = '<h1>Welcome to Authors Haven API Version 1.0</h1> ';
   response.status(200).send(welcome);
```

Comment:
Let's use the http helper here #consistency

Reply:
@seyi-adeleke noted.